install/kubernetes/cilium/templates/hubble-relay/configmap.yaml

```diff
@@ -7,7 +7,12 @@ metadata:
   namespace: {{ .Release.Namespace }}
 data:
   config.yaml: |
+    cluster-name: {{ .Values.cluster.name }}
+    {{- if and .Values.hubble.enabled .Values.hubble.listenAddress .Values.hubble.peerService.enabled }}
+    peer-service: "hubble-peer.{{ .Release.Namespace }}:{{ .Values.hubble.peerService.servicePort }}"
```

Comment:
Does this address work in a cluster mesh scenario? I'm wondering if we don't need to specify `.svc.cluster-domain.example` there.